File: 1967. Number of Strings That Appear as Substrings in Word.js
// 1967. Number of Strings That Appear as Substrings in Word
patterns = ["a","abc","bc","d"]
word = "abc"
// SOL 1
function num_of_strings(patterns, word) {
  return patterns.filter(p => word.includes(p)).length;
}

console.log(num_of_strings(patterns, word))

File: 1967. Number of Strings That Appear as Substrings in Word.py
#* 1967. Number of Strings That Appear as Substrings in Word
patterns = ["a","abc","bc","d"]
word = "abc"
# @param {String[]} patterns
# @param {String} word
# @return {Integer}
#* SOL 1
def num_of_strings(patterns, word):
    return sum(1 for p in patterns if p in word)

print(num_of_strings(patterns, word))